packages/elements/src/internals/machines/sign-up/router.machine.ts

```diff
@@ -343,8 +344,10 @@ export const SignUpRouterMachine = setup({
       },
       on: {
         'RESET.STEP': {
-          target: 'Start',
-          reenter: true,
```

Comment:
This was causing the child machine to be re-created, and subsequently it would move right back to `Attempting` because of the presence of the ticket. For `RESET.STEP` specifically, we want to update the `formRef` of the child machine as this even is fired when the form machine is re-created.

Reply:
There might be a better spot to put this that makes the intent more explicit. The underlying attempt actor was getting canceled, and subsequently re-created, by the reinitialization of the start machine. Changing this to only updating the form ref (which is what triggers `RESET.STEP`) solved the re-initialization issue.